src/Runner.Common/ExtensionManager.cs

```diff
@@ -56,6 +56,10 @@ public sealed class ExtensionManager : RunnerService, IExtensionManager
                     Add<T>(extensions, "GitHub.Runner.Worker.EndGroupCommandExtension, Runner.Worker");
                     Add<T>(extensions, "GitHub.Runner.Worker.EchoCommandExtension, Runner.Worker");
                     break;
+                case "GitHub.Runner.Worker.IFileCommandExtension":
```

Comment:
nit: personally don't like extension :P

nit: technically not "commands". Wonder if different naming would be more intuitive (less overloading)